universe/interface.go

```diff
@@ -673,8 +673,16 @@ type AssetSyncStats struct {
 
 // AggregateStats is a set of aggregate stats for a given Universe.
 type AggregateStats struct {
-	// NumTotalAssets is the total number of assets in the Universe.
-	NumTotalAssets uint64
+	// NumTotalAssetUnits is the total number of asset units in the
+	// Universe.
+	NumTotalAssetUnits uint64
+
+	// NumTotalAssetTranches is the total number of asset tranches in the
+	// Universe.
+	NumTotalAssetTranches uint64
```

Comment:
How's this differ from groups? This is the total amount of issued asset UTXOs for each group?